package.json

```diff
@@ -31,11 +31,11 @@
     "assets-webpack-plugin": "7.0.*",
     "babel-loader": "^9.1.3",
     "chalk": "^4.0.0",
-    "clean-webpack-plugin": "^4.0.0",
     "css-loader": "^6.7.0",
     "css-minimizer-webpack-plugin": "^5.0.0",
     "fastest-levenshtein": "^1.0.16",
     "mini-css-extract-plugin": "^2.6.0",
+    "multimatch": "^5",
```

Comment:
Can we use [picomatch](https://www.npmjs.com/package/picomatch) instead?

It has 0 dependencies (where `multimatch` has 7 dependencies), and weight much less, see graphs:
- picomatch: https://npmgraph.js.org/?q=picomatch#select=exact%3Apicomatch%404.0.2
- mulitmatch: https://npmgraph.js.org/?q=multimatch%406.0.0

picomatch is also compatible with CJS modules.

Thanks!

Reply:
and picomatch might switch to ESM in the future as well (there is a PR opened about that). So just using picomatch won't let us use CJS forever (my expectation is that more and more packages will migrate to ESM)

Reply:
note that this still reduces our dependencies compared to `clean-webpack-plugin`

Reply:
What about breaking the cleaning feature and let the user decide what it wants to **keep** (through `output.clean.keep` if I remember)? 
We could directly pass its configuration to Webpack, it will be easier for us, no need for extra work nor dependencies.
WDYT ?

Reply:
We still need the config to keep the manifest.json for the dev-server (unless the issue mentioned in the code comment is not relevant anymore.

If we can remove that part, I'm fine with doing a bigger BC break dropping the first argument (and letting users configure `keep` through the callback if needed).

Reply:
However, as I'm not using the dev-server myself (I'm using the watch mode), I don't have a setup to test this.

Reply:
I can give it a try tomorrow

Reply:
Good news, the option `output.clean.keep()` seems totally ignored in dev-server:
- Just to be sure, I've removed by `public/build` folder, 
- Commented `cleanedPaths.push('!manifest.json')`
- Ran the dev-server

And file `public/build/manifest.json` is present, with good URLs, HMR works too, etc... :)

So we can remove `cleanedPaths.push('!manifest.json')` and go for a bigger BC :)

Reply:
I updated the PR with the simpler implementation.